### PR TITLE
Add concept about Grafana and Grafana Enterprise database encryption

### PR DESCRIPTION
**Pull request by osg-grafana**
Partial documentation for https://github.com/grafana/grafana-enterprise/issues/2196.

### CODE REVIEW

**Comment by joanlopez** [comment]
Since the v8.3. release is going to happen now (10AM CET), we may need to add the backport label, just wondering.

File: docs/sources/administration/database-encryption.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title = "Database encryption"
+description = "Grafana database encryption"
+keywords = ["grafana", "database", "encryption", "documentation"]
+aliases = [""]
+weight = 450
++++
+
+# Grafana database encryption
+
+Grafana’s database contains secrets, which are used to query data sources, send alert notifications and perform other functions within Grafana:
```
Comment (osg-grafana) [inline docs/sources/administration/database-encryption.md:11]:
```suggestion
Grafana’s database contains secrets, which are used to query data sources, send alert notifications and perform other functions within Grafana.
```

File: docs/sources/administration/database-encryption.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (osg-grafana) [inline docs/sources/administration/database-encryption.md:12]:
```suggestion
```

File: docs/sources/administration/database-encryption.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* **Data source credentials:** GitHub API tokens or MySQL authorization credentials
```
Comment (osg-grafana) [inline docs/sources/administration/database-encryption.md:13]:
```suggestion
```

File: docs/sources/administration/database-encryption.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+* **Notification channel credentials:** email credentials or PagerDuty integration keys
```
[the remark this answers is not in the document]
Comment (osg-grafana) [reply docs/sources/administration/database-encryption.md:14]:
```suggestion
```

File: docs/sources/administration/database-encryption.md (hunk 2 above)
Comment (osg-grafana) [inline docs/sources/administration/database-encryption.md:12]:
```suggestion
```

**Review by undef1nd — APPROVED** [review]

**Pull request merged**